internal/wasmdebug/dwarf_test.go

```diff
@@ -82,109 +82,72 @@ func TestDWARFLines_Line_Rust(t *testing.T) {
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/fmt/mod.rs:1407:36
-	//          10: 0xb1a3 - <core::panic::location::Location as core::fmt::Display>::fmt::hf3870c0af6a67fac
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/panic/location.rs:196:6
-	//          11: 0x8ee0 - <unknown>!std::rt::lang_start_internal::h3c39e5d3c278a90f
-	//          12: 0xae9d - core::fmt::write::h5471a2341ce22f17
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/fmt/mod.rs:1226:2
-	//          13: 0xc3b7 - core::char::methods::encode_utf8_raw::h700e7a293d6eb2b7
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/char/methods.rs:1677:13              - core::char::methods::<impl char>::encode_utf8::h641f2d1d001008d5
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/fmt/mod.rs:165:24              - core::fmt::Write::write_char::ha951e2975b9730c3
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/fmt/mod.rs:165:24
-	//          14: 0xc418 - core::fmt::Write::write_fmt::h4a7e084f8beacf08
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/fmt/mod.rs:187
-	//          15: 0xc6e7 - core::iter::traits::iterator::Iterator::fold::h99ed29c108afc948
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/iter/traits/iterator.rs:2414:21              - <core::iter::adapters::map::Map<I,F> as core::iter::traits::iterator::Iterator>::fold::ha5be3bb1eeeaf8fe
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/iter/adapters/map.rs:124:9              - <usize as core::iter::traits::accum::Sum>::sum::h00b4d0c0300e94a9
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/iter/traits/accum.rs:42:17              - core::iter::traits::iterator::Iterator::sum::h04374b17d4abbea5
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/iter/traits/iterator.rs:3347:9              - <core::iter::adapters::filter::Filter<I,P> as core::iter::traits::iterator::Iterator>::count::h5c8b4c5e67e6831c
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/str/count.rs:135:5              - core::str::count::char_count_general_case::hfffa06842344b9fe
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/str/count.rs:135:5              - core::str::count::do_count_chars::h124622240ac1fb8b
-	//                           at /rustc/c396bb3b8a16b1f2762b7c6078dc3e023f6a2493/library/core/src/str/count.rs:71:21
+	// 0: failed to invoke command default
```

Comment:
Was confused but AFAICT, this test case / comment was created from a different binary than what we have in the source, so I tried updating it.

https://github.com/tetratelabs/wazero/blob/main/internal/testing/dwarftestdata/testdata/rust/main.rs